### ISSUE TITLE
Fail to start api and web 

### ISSUE THREAD

**Issue by Menghua1** [Azure/azure-dev#4107]
**Describe the issue:**
- PowerShell - Web error

When the default terminal is `PowerShell`, starting the api succeeds, but **starting the web fails** and the terminal reports the following error message(all todo templates have this issue ):
![image](https://github.com/Azure/azure-dev/assets/111940661/742905bd-fea5-4ee5-9c32-6e23b453ecfb)
- Bash - API error 

When changing the default terminal to `Bash`, starting the web succeeds, but **starting the api fails** and the terminal reports the following error message (only `todo-python-mongo` have this issue ):
![image](https://github.com/Azure/azure-dev/assets/111940661/753337fe-7f22-4025-9362-7d269743faa8)
**Repro Steps:**
1. Login in with command: `azd auth login`.
2. Execute command: `azd init -t todo-python-mongo -b staging`.
3. Run `azd up`.
4. Hit F1 -> Run Task -> Start API and Web.

**Environment:**

- Template: `todo-python-mongo`
- Branch: `pr/3976` and `staging`
- OS: Windows
- Azd version: `azd version 1.10.0-beta.1-daily.3947259 (commit 5bf8568b1566723be8bc2259c578df4f931cf430)`

**Expected behavior:**
Open the default terminal `poweshell` or `bash`, both api and web can be started successfully.

@jongio for notification.

**Comment by jongio** [comment]
This may have been transient.  Please re-test

**Comment by v-xuto** [comment]
> This may have been transient. Please re-test

Currently, we are blocked by issue #4019 and unable to re-test. We will re-test it after issue #4019 is resolved.

**Comment by jongio** [comment]
@rajeshkamal5050 or @weikanglim @vhvb1989 Can you follow up?

**Comment by vhvb1989** [comment]
using `cross-env` to set the ENV VAR could work here.
I can take this

**Comment by Menghua1** [comment]
@vhvb1989 We have retested this issue, you only solved issue1, not issue2. 
For `todo-python-mongo` in Windows, when the default terminal is `bash`, `start api` still has the following issue. When the default terminal is `pwsh`, `start api and web` are fine.
![image](https://github.com/user-attachments/assets/aa014cbe-926a-43bd-9f51-e3d13edeb235)

**Comment by jongio** [comment]
> @vhvb1989 We have retested this issue, you only solved issue1, not issue2. For `todo-python-mongo` in Windows, when the default terminal is `bash`, `start api` still has the following issue. When the default terminal is `pwsh`, `start api and web` are fine. ![image](https://private-user-images.githubusercontent.com/111940661/348592970-aa014cbe-926a-43bd-9f51-e3d13edeb235.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEwNTIxNTgsIm5iZiI6MTcyMTA1MTg1OCwicGF0aCI6Ii8xMTE5NDA2NjEvMzQ4NTkyOTcwLWFhMDE0Y2JlLTkyNmEtNDNiZC05ZjUxLWUzZDEzZWRlYjIzNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxNVQxMzU3MzhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hYmRiM2VlOTAxMDZkNzhiNDAwMDcwMTliZGE5OWI1N2QyYzNmOWEyYzFlNjBlYmI0NjAyNDQ2NDBjNGY0YzdmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.U7fv-91rVBcy5RxHNcxgyIXje1DMVz2FIhpbRp2d8C8)

@Menghua1 
Can you please open a new issue for this?  Thanks

**Comment by Menghua1** [comment]
> > @vhvb1989 We have retested this issue, you only solved issue1, not issue2. For `todo-python-mongo` in Windows, when the default terminal is `bash`, `start api` still has the following issue. When the default terminal is `pwsh`, `start api and web` are fine. ![image](https://private-user-images.githubusercontent.com/111940661/348592970-aa014cbe-926a-43bd-9f51-e3d13edeb235.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEwNTIxNTgsIm5iZiI6MTcyMTA1MTg1OCwicGF0aCI6Ii8xMTE5NDA2NjEvMzQ4NTkyOTcwLWFhMDE0Y2JlLTkyNmEtNDNiZC05ZjUxLWUzZDEzZWRlYjIzNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxNVQxMzU3MzhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hYmRiM2VlOTAxMDZkNzhiNDAwMDcwMTliZGE5OWI1N2QyYzNmOWEyYzFlNjBlYmI0NjAyNDQ2NDBjNGY0YzdmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.U7fv-91rVBcy5RxHNcxgyIXje1DMVz2FIhpbRp2d8C8)
> 
> @Menghua1 Can you please open a new issue for this? Thanks

Ok, new issue is https://github.com/Azure/azure-dev/issues/4127.
